src/app/preview-area/preview/preview.component.html

```diff
@@ -0,0 +1 @@
+<iframe #iframe height="719px" width="395px"></iframe>
```

Comment:
How did you decide on these pixel values? Does this correspond to a certain phone HeightxWidth?

Reply:
I just set it to be the size that looked best on my monitor. Planning to change it so it's responsive instead of static.